net/wsdd2/Makefile

```diff
@@ -5,9 +5,9 @@ PKG_RELEASE:=3
 
```

Comment:
PKG_RELEASE needs to be resetted.

Reply:
I'm not sure what this means 
if it's about the build system 
I erased everything 
update and downloaded feeds
change date & version only NOT Hash
then compiled 
it downloads new file into dl & compiles ok with out errors at all 
I could not get it to fail :(

Reply:
I said that you should change only the value from 3 into 1. Dont reinvent a wheel, dude.

Reply:
ok thanks will do